conans/client/cmd/export.py

```diff
@@ -246,6 +246,10 @@ def _classify_patterns(patterns):
             excluded.append(p[1:])
         else:
             included.append(p)
+
+    # FIXME: This is weird, as the test_package folder could be named different... do we want it?
```

Comment:
So, I would say, better to be explicit and remove this.

Reply:
Yes, agree. Users should be careful and not ``export`` the ``test_package`` folder. The only problem is that it might not be very intuitive on the UI. The ``export()`` method can do a controlled copy, and use ``FileCopier.copy(exclude=...)`` or something like that, but with the attribute ``exports = "*"``, that will be bad. But yes, lets remove it. Also we have harcoded ``if basename in [".git", ".svn"]:`` to exclude those subfolders, which are annoying in the 99% of cases to export. But some users asked sometime ago how to export the .git repo folder.